src/Electron/electron.ts

```diff
@@ -243,7 +244,7 @@ export class ElectronContainer extends WebContainerBase {
     public createWindow(url: string, options?: any): ContainerWindow {
         const newOptions = this.getWindowOptions(options);
         const electronWindow: any = new this.browserWindow(newOptions);
-        const windowName = newOptions.name || url;
+        const windowName = newOptions.name || Guid.newGuid();
```

Comment:
This was changed for OpenFin a while back when name became a required option but we never changed Electron to be consistent.